addons/mail/static/src/components/activity_box/activity_box.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.ActivityBox" owl="1">
         <div class="o_ActivityBox">
             <t t-if="chatter and chatter.thread">
-                <a role="button" class="o_ActivityBox_title btn" t-on-click="chatter.onClickActivityBoxTitle">
+                <a role="button" class="o_ActivityBox_title btn" t-att-aria-expanded="chatter.isActivityBoxVisible ? 'true' : 'false'" t-on-click="chatter.onClickActivityBoxTitle">
```

Comment:
href not added here in this version?

Reply:
You're right. This was a rebase issue, done.